usaspending_api/awards/tests/unit/test_transaction_delta.py

```diff
@@ -0,0 +1,67 @@
+from django.db.utils import IntegrityError
+from django.test import TransactionTestCase
+from model_mommy import mommy
+from time import sleep
+from usaspending_api.awards.models import TransactionDelta
+from usaspending_api.awards.models.transaction_delta import CHUNK_SIZE
+
+
+class TransactionDeltaTestCase(TransactionTestCase):
+
+    def setUp(self):
+        for _id in range(1, 5):
+            mommy.make('awards.TransactionNormalized', id=_id)
+            mommy.make('awards.TransactionFPDS', transaction_id=_id)
+            mommy.make('awards.Award', id=_id, latest_transaction_id=_id)
+
+    def test_stuff(self):
```

Comment:
Better title needed. I would rather see this split out into a couple tests with more precise names.

Reply:
My apologies as this was not intended to be the final version.  I checked in my work in progress just in case I suffered a fatal skydiving accident while on vacation.

Reply:
No such luck.